tools/webots.py

```diff
@@ -178,7 +182,7 @@ def exec_build(target, roles, clean, jobs):
 
     # Build the image!
     exit_code = subprocess.run(
-        ["docker", "build", "-t", f"{ROBOCUP_IMAGE_NAME}:{ROBOCUP_IMAGE_TAG}", "-f", "./nugus_sim.Dockerfile", "."]
+        ["docker", "build", "-t", f"{ROBOCUP_IMAGE_NAME}:{ROBOCUP_IMAGE_TAG}", "-f", "./Dockerfile.nugus_sim", "."]
```

Comment:
Isn't this wrong? we have a file called `nugus_sim.Dockerfile` and we don't have a file named `Dockerfile.nugus_sim`?